docker-compose.yaml

```diff
@@ -49,6 +50,31 @@ services:
     volumes:
       - sentence-transformers:/app/.sentence_transfomers
 
+  # Bring up the llm-gateway service. Requires the machine-learning repo to be cloned in an
+  # adjacent directory. Set your OpenAI key in the .env file (see .env.example for format)
+  llm-gateway:
+    build:
+      context: ../machine-learning/
+      dockerfile: ../machine-learning/mlapi/Dockerfile
+      target: development
+      cache_from:
+        - us.gcr.io/kubernetes-dev/mlops-api:latest
+      args:
+        BUILDKIT_INLINE_CACHE: '1'
+    ports:
+      - 4033:4033
+    working_dir: /go/src/mlapi
+    entrypoint:
+      - 'air'
+    command:
+      - '-c'
+      - '.air.llm-gateway.toml'
+    environment:
+      air_wd: /go/src/mlapi
+      OPENAI_API_KEY: '$OPENAI_API_KEY'
+      OPENAI_ORGANIZATION_ID: '$OPENAI_ORGANIZATION_ID'
+    restart: unless-stopped
+
```

Comment:
1. This repo is open source and others will not be able to do this.
2. The plugin will not be directly interacting with the llm-gateway, first it will go through the Grafana Cloud auth endpoint.

For local dev perhaps just directly use a Grafana Cloud deployment of llm-gateway specified via environment variables?